### PR TITLE
RUN-3797 Fix some tslint issues

### PR DESCRIPTION
**Pull request by licui3936**
This PR is fixing some tslint issues.

Test results: [Windows 7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5a7322633f32861e82b77cfb)  [Windows 10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5a73233b3f32861e82b77cfd)

### CODE REVIEW

**Comment by licui3936** [comment]
I am looking into it and see if i can remove this restriction.

**Comment by licui3936** [comment]
Sim, i fix the rest tsline issue and remove the restriction you mentioned.

File: src/modules.d.ts [hunk 1]
```diff
@@ -109,3 +109,9 @@ declare module 'electron' {
         export function readText(type?: string): string;
     }
 }
+
+declare namespace Rx {
```
Comment (whyn07m3) [inline src/modules.d.ts]: Why do we need this if you added Rx's types in `package.json`?

Comment (licui3936) [reply src/modules.d.ts]: Because scheduleFuture function is missing in rx type definition.  This it the patch.

**Pull request merged**